config/software/gdbm.rb

```diff
@@ -44,6 +44,7 @@
 
   configure_command = ["./configure",
```

Comment:
Could we configure with `--disable-static`? We don't want to ship static libraries, so let's ensure explicitly they're not built.

Reply:
Sure